ui/app/components/ui/circle-icon/index.scss

```
@@ -0,0 +1,27 @@
+.circle-icon {
+  &__container {
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    img {
+      z-index: 1;
+    }
+  }
+
+  &__border {
+    border-radius: 50%;
+    position: absolute;
+  }
+
+  &__circle {
+    border: 1px solid;
+    border-color: $black;
+    background: $white;
+  }
+
+  &__icon {
+    z-index: 1;
```

Why is the `z-index` set on the icon? Also, why is it set both here and on line 9?

---

The `z-index` is set so that the icon appears on top of the circle, which can have a coloured background (and defaults to a white background).

You are right that it does not need to be set in two places.

---

Fixed in 3099a26f1

---

We don't need to use `z-index` to ensure it appears on top though. It comes later in the DOM, so it'll be on top as long as it's a positioned element.

e.g.:
```suggestion
   position: relative;
```

I'd like to avoid the use of z-index if possible, as it can lead to unexpected results with multiple elements with z-index values overlap. If we ensure things are stacked according to the DOM order, everything "just works" the way you'd expect them to.

---

done in 47c627a7a